databricks_cli/pipelines/cli.py

```diff
@@ -208,19 +215,47 @@ def reset_cli(api_client, pipeline_id):
 @provide_api_client
 def run_cli(api_client, pipeline_id):
     """
-    Starts the execution of a delta pipelines run by starting the cluster and processing data.
+    [Deprecated] Use the "start" command instead.
+
+    Starts a pipeline run by starting the cluster and processing data.
 
     Usage:
 
     databricks pipelines run --pipeline-id 1234
     """
+    click.echo("Deprecation warning: the \"run\" command is deprecated." +
+               " Use the \"start\" command instead.")
+    _validate_pipeline_id(pipeline_id)
+    PipelinesApi(api_client).start_update(pipeline_id, full_refresh=False)
+    click.echo("Run triggered for pipeline {}.".format(pipeline_id))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Starts a pipeline run.')
+@click.option('--pipeline-id', default=None, type=PipelineIdClickType(),
+              help=PipelineIdClickType.help)
+@click.option('--full-refresh', default=False, type=bool,
+              help='If true, truncates tables and creates new checkpoint' +
+                   ' folders so that data is reprocessed from the beginning.')
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def start_cli(api_client, pipeline_id, full_refresh):
+    """
+    Starts a pipelines run by starting the cluster and processing data.
+
+    Usage:
+
+    databricks pipelines start --pipeline-id 1234 --full-refresh=true
+    """
     _validate_pipeline_id(pipeline_id)
-    PipelinesApi(api_client).run(pipeline_id)
-    click.echo("Run triggered for pipeline {}".format(pipeline_id))
+    PipelinesApi(api_client).start_update(pipeline_id, full_refresh=full_refresh)
+    click.echo("Started an update for pipeline {}.".format(pipeline_id))
```

Comment:
should this message indicate whether it was a full refresh?

Reply:
This should print the update id of the update that was started.

Reply:
done